### PR TITLE
Update Travis-CI build SDK version 27 -> 28, Fix AndroidX Compat

### PR DESCRIPTION
**Pull request by iambmelt**
See title + diff

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -16,13 +16,13 @@ ext {
     appCompatVersion = "1.0.2"
     browserVersion = "1.0.0"
     dexmakerMockitoVersion = "1.4"
-    espressoCoreVersion = "3.1.0"
```
Comment (rpdome) [inline gradle/versions.gradle]: Can you also push this change to android_complete once this PR goes in?

File: common/build.gradle [hunk 2]
```diff
@@ -93,6 +93,7 @@ dependencies {
     testImplementation "org.powermock:powermock-classloading-xstream:$rootProject.ext.powerMockVersion"
 
     // Instrumented dependencies
+    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
```
Comment (rpdome) [inline common/build.gradle]: Can we add this to the version.gradle file? (Most likely we'll need the same thing for ADAL)

Comment (iambmelt) [reply common/build.gradle]: Yep - once this is fully working I'll tune it up for cleanliness

Comment (iambmelt) [reply common/build.gradle]: @rpdome I am going to merge this as-is for now and will take you suggestion as a follow-up PR into `android-complete`

**Pull request merged**